compiler/src/iree/compiler/GlobalOptimization/test/materialize_homogeneous_encodings.mlir

```diff
@@ -0,0 +1,97 @@
+// RUN: iree-opt --split-input-file --iree-global-opt-materialize-homogeneous-encodings %s | FileCheck %s
+
+#map = affine_map<()[s0] -> ((1025 ceildiv s0) * s0 - 1025)>
+#map1 = affine_map<()[s0] -> ((257 ceildiv s0) * s0 - 257)>
+#map2 = affine_map<()[s0] -> ((513 ceildiv s0) * s0 - 513)>
+#map3 = affine_map<()[s0] -> ((1025 ceildiv s0) * s0)>
+#map4 = affine_map<()[s0] -> ((513 ceildiv s0) * s0)>
+#executable_target_embedded_elf_x86_64_ = #hal.executable.target<"llvm-cpu", "embedded-elf-x86_64", {cpu = "cascadelake", cpu_features = "+cmov,+mmx,+popcnt,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+avx,+avx2,+fma,+avx512f,+bmi,+bmi2,+aes,+pclmul,+avx512vl,+avx512bw,+avx512dq,+avx512cd,+avx512vnni,+adx,+clflushopt,+clwb,+cx16,+cx8,+crc32,+f16c,+fsgsbase,+fxsr,+invpcid,+lzcnt,+movbe,+pku,+prfchw,+rdrnd,+rdseed,+sahf,+x87,+xsave,+xsavec,+xsaveopt,+xsaves", data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", native_vector_size = 64 : index, target_triple = "x86_64-unknown-unknown-eabi-elf", ukernels = true}>
```

Comment:
we should try to drop all the extra stuff we don't need here - it makes the tests harder to understand as it's impossible to tell what is meaningful to this pass - for example, I don't think nearly any of this is useful to the test (or if it is, it's certainly not dependent on whether you have the mmx cpu feature, etc)

Reply:
We only need cpu_features in this test. I removed the rest of attributes.